src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -115,7 +115,7 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      * Fills up all the placeholders of this window.
      */
     void fillInnerParts() {
```

Comment:
```suggestion
    void fillInnerPartsPerson() {
```
Perhaps we can rename this method to make the distinction clearer.

Reply:
Also can thoigh it will be reflected in a future commit
And it will also need to change for the internship version